testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,210 @@ QUnit.module('Form', () => {
+                            {
+                                itemType: 'group',
+                                items: [{
+                                    dataField: editorType,
+                                    label: { visible: true },
+                                    editorType,
+                                }],
+                            },
+                        ],
+                    },
+
+                ],
+            });
+
+            const optionLabelVisible = 'items[0].items[0].items[0].label.visible';
+
+            $formContainer
+                .dxForm('instance')
+                .option(optionLabelVisible, false);
+
+            const $componentWrapper = $formContainer.find(`.dx-${componentName}`).parent();
+
+            assert.notOk($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS));
+        });
+
+        test(`${editorType} should not have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when label.visible=false`, function(assert) {
+            const $form = $('#form').dxForm({
+                labelLocation: 'top',
+                items: [
+                    {
+                        itemType: 'group',
+                        items: [
+                            {
+                                itemType: 'group',
+                                items: [{
+                                    dataField: editorType,
+                                    label: { visible: false },
+                                    editorType,
+                                }, ],
+                            },
+                        ],
+                    },
+
+                ],
+            });
+
+            const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+            assert.notOk($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS));
+        });
+
+        ['left', 'right'].forEach(labelLocation => {
+            test(`${editorType} should not have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when the labelLocation=${labelLocation}`, function(assert) {
+                const $form = $('#form').dxForm({
+                    labelLocation,
+                    items: [
+                        {
+                            itemType: 'group',
+                            items: [
+                                {
+                                    itemType: 'group',
+                                    items: [{
+                                        dataField: editorType,
+                                        label: { visible: true },
+                                        editorType,
+                                    }, ],
+                                },
+                            ],
+                        },
+
+                    ],
+                });
+
+                const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+                assert.notOk($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS));
```

Comment:
Seems it will better to use strictEqual($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS), true, 'text')

Reply:
Why, and how it will help in this case to understand?

If we check whether the class exists or not and expect a boolean value

Reply:
I think it strict comparison is better way in most cases for all types. For a strict comparison (===), we use strictEqual(). https://karloespiritu.github.io/cheatsheets/qunit/

Reply:
Thanks